pom.xml

```diff
@@ -37,15 +37,6 @@
                 <role>maintainer</role>
             </roles>
         </developer>
-        <developer>
-            <id>christ66</id>
-            <name>Steven Christou</name>
-            <email>schristou88@gmail.com</email>
-            <roles>
-                <role>release</role>
-                <role>maintainer</role>
-            </roles>
-        </developer>
```

Comment:
Unrelated to the PR, but this person is no maintainer according to nexus permissions.